pkg/cmd/server/kubernetes/master.go

```diff
@@ -97,6 +105,22 @@ func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string
 	recycler.Run()
 }
 
+// attemptToLoadRecycler tries decoding a pod from a filepath for use as a recycler for a volume.
+// If successful, this method will set the recycler on the config.
+// If unsucessful, an error is returned.
+func attemptToLoadRecycler(path string, config *volume.VolumeConfig) error {
+	glog.V(5).Infof("Attempting to load recycler pod file from %s", path)
+	if path != "" {
```

Comment:
what happens if path is empty, no error is returned, and no pod template is set?